cmd/fetchd/cmd/root.go

```diff
@@ -57,12 +58,20 @@ func NewRootCmd() (*cobra.Command, params.EncodingConfig) {
 		WithInput(os.Stdin).
 		WithAccountRetriever(types.AccountRetriever{}).
 		WithBroadcastMode(flags.BroadcastBlock).
-		WithHomeDir(app.DefaultNodeHome)
+		WithHomeDir(app.DefaultNodeHome).
+		WithViper("FETCH")
```

Comment:
this add a `FETCH_`  prefix to env variables, so they don't conflict with other cosmos apps on the machine. It uses `viper.AutomaticEnv` so any flag can be set via env: `FETCH_NODE`, `FETCH_CHAIN_ID`...

The order or precedence for config is: flags > env vars > client.toml file.